### PR TITLE
made changes to counting

### PR DESCRIPTION
**Pull request by ProFireDev**
this should fix that undefined bug

### CODE REVIEW

**Comment by ProFireDev** [comment]
seems like there is come conflicts in the index, should I fix that or?

**Comment by emilianya** [comment]
Yes index conflicts need to be fixed

**Comment by emilianya** [comment]
okay uh.. idk if github is displaying tvings in a weird way but you seem to have entirely replaced the getting of highscorw with lastcounter?!

File: commands/counting.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ module.exports = {
 					con.end();
 				});
 			} else {
-				sql = `SELECT highscore FROM counting WHERE (serverid = ${message.guild.id})`;
```
Comment (emilianya) [inline commands/counting.js:40]: tf?

**Review by emilianya — CHANGES_REQUESTED** [review]
Okay I am denying this review. I am not sure what all the other changes are as I will focus on counting.

1) You shouldn't add last counter to the highscore one
2) You need to **add** last counter, not entirely replace the existing number.

**Comment by emilianya** [comment]
I peeked at the conflicts and had a stroke. Good Luck.

**Comment by ProFireDev** [comment]
> I peeked at the conflicts and had a stroke. Good Luck.

I uh might of fixed those, I hope, we'll see I guess lol. idk why there's so many considering I made zero changes to it

**Comment by ProFireDev** [comment]
reopening this rq

File: commands/ping.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ module.exports = {
 	name: "ping",
 	description: "Ping!",
 	botPermissions: ["EMBED_LINKS"],
-	async execute(message, args) {
+	async execute(message, args) { // no idea, but this works
```
Comment (nyakase) [inline commands/ping.js:7]: uhh what is this comment for

File: commands/pong.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ module.exports = {
 	name: "pong",
 	description: "Pong.",
 	botPermissions: ["EMBED_LINKS"],
-	async execute(message, args) {
+	async execute(message, args) { //removed args for testing - seems to of worked /shrug
```
Comment (nyakase) [inline commands/pong.js:7]: another comment here

File: config.json [hunk 4]
```diff
@@ -67,6 +67,7 @@
         "enabled": true,
         "channelName": "reports",
         "staffRoleName": "Staff"
+
```
Comment (nyakase) [inline config.json:70]: random space?!

File: counting.js [hunk 5]
```diff
@@ -9,7 +9,7 @@ const error = chalk.bold.red;
 const warn = chalk.yellowBright;
 const success = chalk.green;
 var servers = {};
-var cheader = `[${vukkytils.getString("COUNTING")}]`;
+var cheader = `[${vukkytils.getString("")}]`;
```
Comment (nyakase) [inline counting.js:12]: i think you fucked up the entire counting.js file here

**Review by nyakase — CHANGES_REQUESTED** [review]

**Comment by nyakase** [comment]
yeah uh you definitely fucked up on this one
image

File: commands/pong.js (hunk 3 above)
Comment (ProFireDev) [reply commands/pong.js:7]: my random things to remember what I was testing. merging dev into main was a bit of a mistake lol

File: config.json (hunk 4 above)
Comment (ProFireDev) [reply config.json:70]: had to edit the config file to put the user id's back, oops thats gonna get removed

File: counting.js (hunk 5 above)
Comment (ProFireDev) [reply counting.js:12]: yea fucked ups a bit of an under statement, time to copy and paste it back lol

**Comment by emilianya** [comment]
Ever heard of reverting commits

**Comment by ProFireDev** [comment]
> Ever heard of reverting commits

yea I was gonna do that, but I think its gonna be better in the long run, if I just reclone dev and start fresh to make sure there will be no conflicts

**Comment by GamerBoi153** [comment]
@Vukky | Owner Тупой!

**Comment by GamerBoi153** [comment]
Я обошел систему!

**Comment by GamerBoi153** [comment]
Послушай меня!

**Comment by GamerBoi153** [comment]
Обещаю, если вы включите звук. Я не буду говорить на своем родном языке. Хорошо?

**Comment by GamerBoi153** [comment]
Посмотри бревна вуккий мальчик, я с тобой разговариваю!

**Comment by GamerBoi153** [comment]
Я знаю, ты не спишь!

**Comment by GamerBoi153** [comment]
Даже не пытайтесь!

**Comment by GamerBoi153** [comment]
Съешь все сообщения, вкуснятина, да?

**Comment by GamerBoi153** [comment]
Ага. продолжайте удалять их vukky. Покажи свое истинное лицо!

**Comment by emilianya** [comment]
I see this is open, have all the issues been fixed?

**Comment by nyakase** [comment]
i do not remember undrafting this :eyes:

**Comment by ProFireDev** [comment]
> i do not remember undrafting this 👀
I haven't made changes, it's still marked as a draft for me too

**Comment by ProFireDev** [comment]
I have now closed it cuz I fixed all the stuff by pulling from main and then replacing that with dev, so it should be the same as everything before the modals

**Pull request closed without merging**